pyembed/core/discovery.py

```diff
@@ -219,12 +219,16 @@ def __init__(self, url):
     def get_oembed_urls(self, url, oembed_format=None):
         if not self.endpoints:
             response = requests.get(self.url)
+            try:
+                providers = response.json()
+            except ValueError:
+                providers = None
```

Comment:
So the reason why I'm also checking for `or not providers` is because like how it was happening today `response.json` was actually `False`.